src/styles/main.scss

```diff
@@ -0,0 +1,145 @@
+.card-container {
+  width: 100%;
+  box-sizing: border-box;
+  color: #060b35;
+  gap: 46px 48px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  place-content: center;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .card-container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .card-container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .card-container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product-card {
+  display: flex;
+  box-sizing: border-box;
+  place-content: center;
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  transition: all 0.3s;
+}
+
+.product-card:hover {
+  transform: scale(1.2);
```

Comment:
The transform property may cause the card to overflow its container when it is scaled up. You should consider adding 'overflow: hidden' to the parent container to prevent this.